frontend/src/components/story/StoryDay.vue

```diff
@@ -93,10 +98,18 @@ export default {
       return this.entries.filter(({ storyChapters }) => storyChapters.length)
     },
   },
-  mounted() {
-    this.day
-      .scheduleEntries()
-      .items.forEach((entry) => this.api.reload(entry.activity().contentNodes()))
+  async mounted() {
+    // refresh to get new schedule entries
+    await this.day.scheduleEntries().$reload()
+    // refresh individual schedule entries to get new story content nodes
+    await Promise.all(
+      this.day.scheduleEntries().items.map((entry) =>
+        entry
+          .activity()
+          .contentNodes()
+          ._meta.load.then((contentNodes) => this.api.reload(contentNodes))
```

Comment:
This is an n+1 query, but it was one before, so you don't have to fix that in this PR